src/ya-comiste-react-native/src/relay/QueryRenderer.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+
+import * as React from 'react';
+import { SafeAreaView, StyleSheet, ActivityIndicator } from 'react-native';
+import { QueryRenderer as RelayQueryRenderer, type Variables } from 'react-relay';
+
+import relayEnvironment from './relayEnvironment';
+
+type Props = {|
+  +query: any, // TODO
```

Comment:
There should be a `GraphqlTaggedNode` or something exported from relay-runtime I think

Reply:
fixed